package.json

```diff
@@ -96,13 +103,31 @@
         },
         "editor.autoIndent": "keep"
       }
-    }
-  },
-  "configuration": {
-    "yaml.keyOrdering": {
-      "type": "boolean",
-      "default": false,
-      "description": "Enforces alphabetical ordering of keys in mappings when set to true"
+    },
+    "configuration": {
+      "title": "Codecov",
+      "properties": {
+        "codecov.coverage.enabled": {
+          "type": "boolean",
+          "default": "false",
+          "description": "Toggle Codecov line coverage decorations.",
```

Comment:
nit: is decorations the right word? Not too important, just something I spent too much time thinking about lol

Reply:
Hah yes this is what they are referred to in vscode's extension API.